internal/blocktx/processor_test.go

```diff
@@ -176,7 +177,8 @@ func TestHandleBlock(t *testing.T) {
 			var blockRequestCh chan blocktx.BlockRequest = nil
 			blockProcessCh := make(chan *p2p.BlockMessage, 10)
 
-			peerHandler := blocktx.NewPeerHandler(logger, blockRequestCh, blockProcessCh)
+			// when
+			sut := blocktx.NewPeerHandler(logger, blockRequestCh, blockProcessCh)
```

Comment:
I think we are testing `processor` here

Reply:
I think it is kind of both because the title of the test is `TestHandleBlock` so I guess the `HandleBlock()` function is tested here